lib/brainstem/concerns/controller_param_management.rb

```diff
@@ -0,0 +1,20 @@
+module Brainstem
+  module Concerns
+    module ControllerParamManagement
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :brainstem_plural_model_name, :brainstem_model_name,
+                        instance_accessor: false, instance_reader: false, instance_writer: false
+      end
+
+      def brainstem_model_name
+        self.class.brainstem_model_name.to_s.presence || controller_name.singularize
```

Comment:
Hey @einzige.  No, `controller_name` returns something like `tasks`, not `task`.